src/main.rs

```diff
@@ -197,9 +221,13 @@ fn build_author_map(
     for oid in walker {
         let oid = oid?;
         let commit = repo.find_commit(oid)?;
-        let commit_author = Author::new(commit.author());
-        let author = mailmap.canonicalize(&commit_author);
-        author_map.add(author, oid);
+        // let mut commit_authors = commit_coauthors(&commit);
```

Comment:
Looks like this needs to be uncommented?

Reply:
Whoops, fixed.